src/RoadRegistry.BackOffice.Schema/RoadNetworkChange.cs

```diff
@@ -12,22 +8,4 @@ public class RoadNetworkChange
         public string Content { get; set; }
```

Comment:
When moving to nullable reference type enabled projects, this changes from a string NULLable field to a required string field. If this is fine, good, otherwise it needs to change to `string?`

Reply:
Just checked that `b.Property(p => p.Content).IsRequired(false);` results in `Content = table.Column<string>(nullable: true),`